APISpec.md

```diff
@@ -97,3 +97,247 @@ Prerequisites are formatted as follows:
 ]` Means in order to take the course, you must have completed course id `2983` AND have completed `734` OR `1673`.
 
 ## Majors
+
+#### `GET /majors/`
+
+Returns a list of all majors.
+
+#### Response:
+
+```json
+[
+  {
+    "major_id": 34,
+    "name": "Computer Science",
+    "degree": "BS"
```

Comment:
Do we want majors to be tied to degrees like this? What if we instead do something like `"available_degrees": ["BS", "MS"]`?

Reply:
Good idea, since some majors may have multiple degrees, while some only have one.